enterprise/admin/views.py

```diff
@@ -165,7 +166,7 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request)
         return list(emails) + already_linked_emails
 
     @classmethod
-    def _enroll_users(cls, emails, course_details, mode, request):
+    def _enroll_users(cls, enterprise_customer, emails, course_details, mode, request):
         """
         Enroll the users with the given email addresses to the course specified by course_details.
         """
```

Comment:
Since we're here, we should add descriptions for all of the arguments to the docstring for this function.